cegs_portal/search/views/v1/experiment.py

```diff
@@ -31,13 +31,30 @@ def is_archived(self):
         except ObjectNotFoundError as e:
             raise Http404(str(e))
 
+    def request_options(self, request):
+        """
+        GET queries used:
+            exp (multiple)
+                * Should be a valid experiment accession ID
+        """
+        options = super().request_options(request)
+        options["analyses"] = request.GET.getlist("analysis", "vpdata.pd")
+
+        return options
+
     def get(self, request, options, data, exp_id):
+        analysis = ExperimentSearch.analysis_id_search(exp_id)
```

Comment:
If the request includes a valid `analysis` query, then analysis should be that analysis. If not, analysis should be the default analysis for the experiment.